File: Java/Variables/src/Variables.java

public class Variables {

	public static void main(String[] args) {

		int x; //Declaration
		
		//int x = 123; // Initialization
		
		x = 123;  // Assignment
		
		System.out.println(x); //Prints x (123)
		System.out.println("\nThe variable is " +x); // Join functions
		
		//Large numbers
		long studentdebt = 42342333423423434L; // Output large numbers
		System.out.println(studentdebt +" <-- This is a very large number\n");
		
		//Print a decimal
		float decimal = 3.14f;
		System.out.println(decimal +" <-- One method");
		
		//Alternative:
		double otherdecimal = 420.69;
		System.out.println(otherdecimal +" <-- The other method (better)\n");
		
		//Boolean
		boolean bool = true;
		System.out.println(bool);
		boolean bool2 = false;
		System.out.println(bool2 +"\n");
		
		//Characters
		char character = '@';
		System.out.println(character);
		
		//Strings
		String name = "Owen";
		System.out.println("\nHello "+name);
		

	}

}

File: C#/variables/variables.cs
using System;
using System.Security.Principal;

namespace variables;

class variables
{
    static void Main(string[] args)
    {   
        // Just declaration, no value assigned
        //int x;
        
        // Declaration and assignment
        
        // int = whole integer
        int x = 69;
        int y = 420;
        
        Console.WriteLine(x);
        Console.WriteLine(y);

        // double = Decimal
        double pi = 3.14;
        Console.WriteLine(pi);
        
        // bool (true or false)
        bool t = true;
        bool f = false;
        Console.WriteLine(t);
        Console.WriteLine(f);
        
        // 1 single character
        char symbol = '@';
        Console.WriteLine(symbol);
    }
}